databases/week2/music2.sql

```diff
@@ -0,0 +1,94 @@
+
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+
+-- -----------------------------------------------------
+-- Schema music
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `music` DEFAULT CHARACTER SET utf8 ;
+USE `music` ;
+
+-- -----------------------------------------------------
+-- Table `music`.`songs`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `music`.`songs` (
+  `TrackId` INT NOT NULL AUTO_INCREMENT,
+  `SingersId` INT NOT NULL,
+  `AlbumsId` INT NOT NULL,
+  `SongName` VARCHAR(45) NULL,
+  PRIMARY KEY (`TrackId`, `SingersId`, `AlbumsId`))
```

Comment:
add singer_id and album_id as foreign keys with reference to their respected tables